app/src/main/java/org/openobservatory/ooniprobe/activity/CustomWebsiteActivity.java

```diff
@@ -66,18 +66,22 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             RunningActivity.runAsForegroundService(CustomWebsiteActivity.this, suite.asArray(), this::finish);
```

Comment:
```suggestion
            // Condition below is only true if the `loadFromWebConnectivity` has been clicked.
            if (binding.loadFromWebConnectivity.getVisibility() == View.INVISIBLE) {
                suite.getTestList(preferenceManager)[0].setMax_runtime(preferenceManager.getMaxRuntime());
            }
            RunningActivity.runAsForegroundService(CustomWebsiteActivity.this, suite.asArray(), this::finish);
```

@hellais , what do you think about this.